test/integration/transactions.js

```diff
@@ -264,17 +264,26 @@ describe('bitcoinjs-lib (transactions)', function () {
     const txb = new bitcoin.TransactionBuilder()
 
     txb.setVersion(1)
-    txb.addInput('61d520ccb74288c96bc1a2b20ea1c0d5a704776dd0164a396efec3ea7040349d', 0) // Alice's previous transaction output, has 15000 satoshis
+    // Alice's previous transaction output, has 15000 satoshis
+    txb.addInput('61d520ccb74288c96bc1a2b20ea1c0d5a704776dd0164a396efec3ea7040349d', 0)
     txb.addOutput('1cMh228HTCiwS8ZsaakH8A8wze1JR5ZsP', 12000)
     // (in)15000 - (out)12000 = (fee)3000, this is the miner fee
 
-    txb.sign(0, {
+    const fakeECPair = {
+      // publicKey should be a Buffer with the DER encoded public key
       publicKey: Buffer.from('029f50f51d63b345039a290c94bffd3180c99ed659ff6ea6b1242bca47eb93b59f', 'hex'),
+      // sign is a function that takes a 32 byte Buffer, signs the 32 byte Buffer directly (no extra hashing)
+      // And returns a 64 byte Buffer:
+      //   The 1st 32 bytes is the signature r value in big endian
```

Comment:
`first` , `second`? :smiley:

Reply:
Maybe capitalize R/S too?

Reply:
Do we need to mention they are zero padded?

Reply:
lol... if we're going to start a grammar review on every comment in the repo we're gonna be here a while... lol

`head` and `tail`? The head 32 bytes and The tail 32 bytes have the same letter count.

Reply:
> Do we need to mention they are zero padded?

Reply:
true... I'll add that.

Reply:
done, should be fine.